Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/BoundingBoxes/BoundingBoxGizmoHandleAxisToAffectEnum.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX
+{
+    public enum BoundingBoxGizmoHandleAxisToAffect
+    {
+        X,
```

Comment:
initialize your enums with = 0.

Reply:
I'd also stick all your enums in a /BoundingBoxEnums/ folder.

Reply:
they all seem specialized except AxisToAffect